mm2src/coins/utxo/utxo_tests.rs

```diff
@@ -494,7 +494,7 @@ fn test_withdraw_impl_set_fixed_fee() {
     };
     let expected = Some(
         UtxoFeeDetails {
-            amount: "0.1".parse().unwrap(),
+            amount: "0.0245".parse().unwrap(),
```

Comment:
This looks like an error. `UtxoFixed` should be mapped to `FixedPerKb` internally. The fee shouldn't be lower than `0.1` in this case. Could you recheck it, please?